recipes/wavelet_buffer/all/conanfile.py

```diff
@@ -0,0 +1,134 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMakeDeps, CMake, cmake_layout
+from conan.tools.files import get, copy, rm, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+
+import os
+
+required_conan_version = ">=1.50"
+
+
+class WaveletBufferConan(ConanFile):
+    name = "wavelet_buffer"
+    license = "MPL-2.0"
+    description = "An universal C++ compression library based on wavelet transformation"
+    topics = ("compression", "signal-processing", "wavelet")
+    homepage = "https://github.com/panda-official/WaveletBuffer"
+    url = "https://github.com/conan-io/conan-center-index/recipes/wavelet_buffer"
+    default_options = {
+        "cimg/*:enable_fftw": False,
+        "cimg/*:enable_jpeg": False,
+        "cimg/*:enable_openexr": False,
+        "cimg/*:enable_png": False,
+        "cimg/*:enable_tiff": False,
+        "cimg/*:enable_ffmpeg": False,
+        "cimg/*:enable_opencv": False,
+        "shared": False,
+        "fPIC": True,
+    }
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+
+    def requirements(self):
+        self.requires("openblas/0.3.20")
+        self.requires("blaze/3.8")
+        self.requires("libjpeg-turbo/2.1.2")
+        self.requires("cimg/3.0.2")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        get(
+            self,
+            **self.conan_data["sources"][self.version],
+            destination=self.source_folder,
+            strip_root=True,
+        )
+
+    def layout(self):
+        cmake_layout(self)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["CONAN_EXPORTED"] = True
+        tc.generate()
+
+        tc = CMakeDeps(self)
+        tc.generate()
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 20
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "16",
```

Comment:
There's a help to handle the new compiler `msvc` you can see the usage of it in the template https://github.com/conan-io/conan-center-index/blob/ead3630cfda6ecafa276ce82d359a942234cd19d/docs/package_templates/cmake_package/all/conanfile.py#L90

Reply:
It's not clear from docs why special method better that general compilers check

Reply:
Done

Reply:
> It's not clear from docs why special method better that general compilers check

Because the "Visual Studio" compiler is deprecated and removed in Conan 2.0

The helper handles both in 1.x to allow migration and it should require less maintenance here :)